homeassistant/helpers/script.py

```diff
@@ -640,6 +643,8 @@ def __init__(
         self.sequence = sequence
         template.attach(hass, self.sequence)
         self.name = name
+        self.domain = domain
+        self.description = description
```

Comment:
Wonder if we should default it to include the domain?

```suggestion
        self.description = description or f"{domain} script"
```